.gitignore

```diff
@@ -1 +1,3 @@
 __pycache__/
+api_pb2_grpc.py
```

Comment:
if these are included in gitignore how are they a part of this commit?

Reply:
Will be changed in subsequent PR